thirdparty/github.go

```diff
@@ -887,6 +887,23 @@ func GetGithubPullRequestCommits(ctx context.Context, token, owner, repo string,
 	return commits, nil
 }
 
+func GetGithubPullRequestReviews(ctx context.Context, token, owner, repo string, PRNumber int) ([]*github.PullRequestReview, error) {
```

Comment:
`prNumber` instead of `PRNumber`.

Reply:
PRNumber is used extensively in this code so keeping for consistency

Reply:
Can we change them to be camelCase? It should be `prNumber` rather than `PRNumber` because of Go styling rules, which is that camelCase is the accepted variable casing for local variables and function parameters. This style is inconsistent with the rest of the code base.